src/SFA.DAS.LevyTransferMatching.Web/Controllers/SearchFunding.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Controllers
+{
+    [AllowAnonymous]
+    public class SearchFunding : Controller

```

Comment:
We could use a route of, say, `search-funding` for this.

Or `funding`?

Reply:
I prefer search-funding or search-funding-opportunities as I think it makes the Url "nicer" for the user.
Will go with search-funding on the controller and remove the route from the action as per your other comment